crates/starknet/src/constants.rs

```diff
@@ -2,14 +2,20 @@ use std::net::{IpAddr, Ipv4Addr};
 
 use starknet_types::chain_id::ChainId;
 
-pub(crate) const CAIRO_0_ACCOUNT_CONTRACT_PATH: &str = concat!(
+pub const CAIRO_0_ACCOUNT_CONTRACT_PATH: &str = concat!(
     env!("CARGO_MANIFEST_DIR"),
     "/accounts_artifacts/OpenZeppelin/0.5.1/Account.cairo/Account.json"
 );
-
 pub const CAIRO_0_ACCOUNT_CONTRACT_HASH: &str =
     "0x4d07e40e93398ed3c76981e72dd1fd22557a78ce36c0515f679e27f0bb5bc5f";
 
+pub const CAIRO_1_ACCOUNT_CONTRACT_SIERRA_PATH: &str = concat!(
+    env!("CARGO_MANIFEST_DIR"),
+    "/accounts_artifacts/OpenZeppelin/0.7.0/Account.cairo/Account.sierra.json"
+);
+pub const CAIRO_1_ACCOUNT_CONTRACT_SIERRA_HASH: &str =
```

Comment:
If this sierra hash is derived from the CAIRO_1_ACCOUNT_CONTRACT_SIERRA_PATH, why not add a test that asserts that indeed this is the class hash. Simillar to the this test https://github.com/0xSpaceShard/starknet-devnet-rs/blob/10e19ff066fbe7fcfd0dcb303bd110aee77d8d7a/crates/types/src/rpc/contract_class.rs#L263

Reply:
will include that in the tests I'm currently adding.

Reply:
Tests expanded in:
- https://github.com/0xSpaceShard/starknet-devnet-rs/pull/196/commits/9d256d0a29fd113f06ad3ebda841ac0abafbd183
- https://github.com/0xSpaceShard/starknet-devnet-rs/pull/196/commits/eaf22d76b37fc78b9752ee081ae0150716855dcf